src/actions/userActions.js

```diff
@@ -33,11 +34,16 @@ export const updateUserAction = (walletId: string, field: Object, callback?: Fun
         type: UPDATE_USER,
         payload: { user: updatedUser, state: REGISTERED },
       });
+      Answers.logCustom('Create User');
```

Comment:
`updateUserAction` is used to update user's profile information.
To track new created users `finishRegistration` or `registerOnBackendAction` (could be found in `onboardingActions.js`) might be more relevant :)
Or the message itself should be changed into a more suitable one. Like 'User info updated'.

Reply:
You're right, I've changed this one to track the profile update, and added the create user on `registerOnBackendAction`.